docs/notebooks/tutorial.ipynb

```diff
@@ -161,7 +167,7 @@
     }
    ],
    "source": [
-    "!run_algo.py -f ../../zipline/examples/buyapple.py --start 2000-1-1 --end 2014-1-1 -o buyapple_out.pickle"
+    "!zipline run -f ../../zipline/examples/buyapple.py --start 2000-1-1 --end 2014-1-1 -o buyapple_out.pickle"
```

Comment:
Not sure - I think the intent is to shell out to run that file, which is the case I'm translating.  Do you think we should change this notebook to use the magic here instead?